Cookbook/Technical-Documents/DebugBuilder.md

```diff
@@ -0,0 +1,96 @@
+Debug Builder
+======================
+
+[\[CNSMR\-1602\] Add \`DebugAuthenticatedRootBuilder\` for faster UI debugging iteration by inamiy · Pull Request \#7645 · Babylonpartners/babylon\-ios](https://github.com/Babylonpartners/babylon-ios/pull/7645)
+
+Above PR introduces `DebugAuthenticatedRootBuilder` that allows to directly show a target view builder after authentication.
+
+This feature improves overall UI debugging iteration speed.
+
+## How to make a debug-builder
+
+1. Add a new case in `enum DebugBuilderPattern`
+2. Create a (mock) builder in `DebugAuthenticatedRootBuilder`
+3. Update `Settings.bundle/Root.plist` by adding a new case key
```

Comment:
Need to also update `Babylon/Brand/babylon/Settings.bundle/Root.plist`

Reply:
Updated in 2a1b5da .